recipe/gen_patch_json.py

```diff
@@ -624,6 +624,9 @@ def _gen_new_index(repodata, subdir):
         if any(dep.startswith("libarchive >=3.3") for dep in deps):
             _pin_looser(fn, record, "libarchive", upper_bound="3.6.0")
 
+        if any(dep == "libflang" or dep.startswith("libflang >=5.0.0") for dep in deps) and record.get('timestamp', 0) < 1611789153000:
```

Comment:
Should we add a comment for what this timestamp maps to to aid future historians?

Reply:
```suggestion
        # fix only packages built before the run_exports was corrected.
        if any(dep == "libflang" or dep.startswith("libflang >=5.0.0") for dep in deps) and record.get('timestamp', 0) < 1611789153000:
```